data/tcr_epitope_binding/meta.yaml

```diff
@@ -0,0 +1,44 @@
+name: tcr_epitope_binding
+description: |-
+  T-cells are an integral part of the adaptive immune system, whose survival, proliferation, activation
+          and function are all governed by the interaction of their T-cell receptor (TCR) with immunogenic peptides (epitopes).
+          A large repertoire of T-cell receptors with different specificity is needed to provide protection against a wide range of pathogens.
+          This new task aims to predict the binding affinity given a pair of TCR sequence and epitope sequence.
+targets:
+- id: binding
+  description: TCR epitope binding.
+  units: ''
+  type: binary classification
+  names:
+  - tcr binding affinity
+  - binding affinity
+  - binding
+identifiers:
+- id: epitope_smiles
+  type: SMILES
+  description: 'epitope smiles '
+- id: epitope_aa
+  type: amino acid
+  description: epitope amino acid sequence
+- id: tcr_aa
+  type: amino acid
+  description: tcr amino acid sequence
```

Comment:
Do I understand the dataset correctly that the binding only makes sense if we specify both the TCR and the Epitope?

Reply:
That is right.  Given the epitope and TCR, predict if the pair binds.

Reply:
In this case, we will need to add templates to sample this data correctly. There are examples for the templates in the [Contribution Guide](https://github.com/OpenBioML/chemnlp/blob/main/CONTRIBUTING.md). Let me know if you want some hand with this

Reply:
@kjappelbaum Thanks for the feedback, I attempted to add a template.  However, I am not sure if I fully understand what to do here.  Can you please have a look and provide some help on this?